bitnami/mongodb/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: mongodb
-version: 9.2.3
+version: 9.2.4
```

Comment:
This should be a minor change

Reply:
as in 9.2.3?

Reply:
Hi! No, my colleague is refering to bump the minor version of the char following [semver](https://semver.org/) standards. In this case it would be `9.3.0`